archetypes/oci-mp/src/main/resources/pom.xml.mustache

```diff
@@ -0,0 +1,52 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://maven.apache.org/POM/4.0.0"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.applications</groupId>
+        <artifactId>helidon-applications</artifactId>
+        <version>{{helidonVersion}}</version>
+        <relativePath/>
+    </parent>
+    <groupId>{{groupId}}</groupId>
+    <artifactId>{{artifactId}}</artifactId>
+    <packaging>pom</packaging>
+    <version>{{version}}</version>
+    <name>{{name}}</name>
+
+    <properties>
+        <version.plugin.jandex>1.1.0</version.plugin.jandex>
```

Comment:
same with these.

Reply:
Since we are depending on application pom and not se/mp, we have to define this explicitly.